components/recurring-contributions/UpdateOrderPopUp.js

```diff
@@ -0,0 +1,327 @@
+import React, { Fragment, useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { useMutation, useQuery } from '@apollo/react-hooks';
+import themeGet from '@styled-system/theme-get';
+import gql from 'graphql-tag';
+import { first, get, startCase } from 'lodash';
+import { withRouter } from 'next/router';
+import { defineMessages, FormattedMessage, useIntl } from 'react-intl';
+import styled from 'styled-components';
+
+import { formatCurrency } from '../../lib/currency-utils';
+import { getErrorFromGraphqlException } from '../../lib/errors';
+import { API_V2_CONTEXT, gqlV2 } from '../../lib/graphql/helpers';
+
+import FormattedMoneyAmount from '../FormattedMoneyAmount';
+import { Box, Flex } from '../Grid';
+import LoadingPlaceholder from '../LoadingPlaceholder';
+import StyledButton from '../StyledButton';
+import StyledHr from '../StyledHr';
+import StyledInputAmount from '../StyledInputAmount';
+import StyledRadioList from '../StyledRadioList';
+import StyledSelect from '../StyledSelect';
+import { P } from '../Text';
+
+const TierBox = styled(Flex)`
+  border-top: 1px solid ${themeGet('colors.black.300')};
+`;
+
+const messages = defineMessages({
+  cancel: {
+    id: 'actions.cancel',
+    defaultMessage: 'Cancel',
+  },
+  update: {
+    id: 'subscription.updateAmount.update.btn',
+    defaultMessage: 'Update',
+  },
+  save: {
+    id: 'save',
+    defaultMessage: 'Save',
+  },
+  updateTier: {
+    id: 'subscription.menu.updateTier',
+    defaultMessage: 'Update tier',
+  },
+});
+
+const updateOrderMutation = gqlV2/* GraphQL */ `
+  mutation updateOrderTierOrAmount($order: OrderReferenceInput!, $amount: Int, $tier: TierReferenceInput) {
+    updateOrder(order: $order, amount: $amount, tier: $tier) {
+      id
+    }
+  }
+`;
+
+const getTiersQuery = gql`
```

Comment:
Can't we have that in GraphQL v2?

Reply:
Currently Tiers isn't implemented on Account/Collective in v2 but happy to update that in a separate PR (like I am doing with the payment methods)